src/js_native_api.h

```diff
@@ -89,6 +89,13 @@ NAPI_EXTERN napi_status napi_create_string_utf16(napi_env env,
 NAPI_EXTERN napi_status napi_create_symbol(napi_env env,
                                            napi_value description,
                                            napi_value* result);
+NAPI_EXTERN napi_status napi_create_function_with_length(napi_env env,
+                                             const char* utf8name,
+                                             size_t length,
+                                             napi_callback cb,
+                                             size_t params,
```

Comment:
`params` might mean populated parameter list. What about `argc` or `params_length`, or `arguments_length`?

Reply:
Thank you for your feedback. Before to go forward I will wait other suggestions and reviews.

Reply:
+1 to `arguments_length`

Reply:
+1 to arguments_length

Reply:
+1 to arguments_length